src/components/Layout/RightSidebar.tsx

```diff
@@ -321,6 +321,34 @@ const RightSidebar = () => {
                 </Tooltip>
               </a>
             ))}
+            <a
+              href={`${location.pathname.replace(/\/$/, '')}/index.md`}
+              className="flex h-5 ui-theme-dark group/markdown-link cursor-pointer"
+              onClick={(e) => {
+                // In development mode, markdown files aren't generated, so show alert immediately
```

Comment:
Let's get rid of this comment, the code is descriptive enough

Reply:
Fair